preview-maker.js

```diff
@@ -196,6 +186,12 @@ const fill = (rootNode, color) => {
   innerFill(rootNode);
 };
 
+const isNotGrey = (hsl) =>
```

Comment:
Avoid negation in function names. If I wanted to check is color _is_ gray using this function, I'd have to write `!isNotGray(..)` which is obviously hard to read. Remove the negation and negate the returned boolean.